crates/biome_js_analyze/tests/specs/nursery/useAnchorHref/invalid-2.jsx.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: invalid-2.jsx
+---
+# Input
+```jsx
+<a target="_blank">External</a> 
+```
+
+# Diagnostics
+```
+invalid-2.jsx:1:1 lint/nursery/useAnchorHref ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! <Emphasis>a</Emphasis> elements must have an <Emphasis>href</Emphasis> attribute to ensure accessibility and proper navigation.
```

Comment:
You probably didn't check the snapshots, but as you can notice, they are wrong :)